src/program.js

```diff
@@ -195,5 +196,5 @@ Example: $0 --help run.
       },
     });
 
-  return program.run(runOptions);
+  return program.run(runOptions, absolutePackageDir);
```

Comment:
since `absolutePackageDir` is a required argument, I'd like to see it come before the optional arguments since one might call it without options and `run(undefined, absolutePackageDir)` would look weird.